turbinia/workers/__init__.py

```diff
@@ -18,16 +18,21 @@
 import json
 import logging
 import os
+import platform
 import time
 import uuid
 
```

Comment:
I notice that we don't use u'' for strings in the code base. We should consider doing:
from __future__ import unicode_literals

To make all strings unicode by default. What do you think?